app/controllers/rosters_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class RostersController < ApplicationController
+  before_action :ensure_student_identifier_flipper_is_enabled, :set_organization
+
+  def new
+    @roster = Roster.new
+  end
+
+  def create
+    @roster = Roster.new(identifier_name: params[:identifier_name])
+    @roster.save!
+
+    add_identifiers_to_roster
+
+    @organization.roster = @roster
+    @organization.save!
+
+    flash[:success] = 'Your classroom roster has been saved! Manage it HERE' # TODO: ADD LINK TO MANAGE PAGE
```

Comment:
The TODO here is addressed in the next PR (the manage feature doesn't exist in this commit). This is being staffshipped, so I'm pretty sure this is OK.